roles/ceph-common/meta/main.yml

```diff
@@ -10,3 +10,4 @@ dependencies:
     repos:
       - repo: 'deb {{ apt_repos.ceph.repo }}{{ ceph.stable_release }}/ {{ ansible_lsb.codename }} main'
         key_url: '{{ apt_repos.ceph.key_url }}'
+  - role: serverspec
```

Comment:
Lets leave this out. Changes are coming that will remove this requirement in each role.